Options.py

```diff
@@ -785,6 +785,7 @@ class VerifyKeys(metaclass=FreezeValidKeys):
     verify_item_name: bool = False
     verify_location_name: bool = False
     value: typing.Any
+    display_valid_keys: bool = False  # Should the full list of keys appear in templates
```

Comment:
I feel like this should probably be more specific, like `display_valid_keys_in_template` to match the name Visibility uses and so that people don't think it also applies to WebHost